_docs/for-developer/postback function.md

```diff
@@ -76,6 +77,7 @@ event.transaction.lineItems.product.categoryId |	STRING |	Listing category ident
 event.transaction.lineItems.product.manufacturer |	STRING |	Manufacturer of the product in the cart
 event.transaction.lineItems.product.skuCode |	STRING |	Stock Keeping Unit - identifier of product variation of the product in the cart
 event.transaction.lineItems.product.stock |	INTEGER |	Remaining product units
+event.nonInteraction | BOOLEAN | Is event passed without interaction with user
```

Comment:
Is event passed without interaction with user ->
Event passed without user interaction